experimental/dds/tree2/src/test/util/utils.spec.ts

```diff
@@ -22,4 +22,11 @@ describe("Utils", () => {
 			assert.equal(capitalize(input), expected);
 		}
 	});
+
+	it("transformObjectMap", () => {
+		assert.deepEqual(
+			transformObjectMap({ a: "b", c: "d" }, (value, key) => `${key}${value}`),
+			Object.assign(Object.create(null), { a: "ab", c: "cd" }),
```

Comment:
Just curious, why is the assignment pattern required here?

Reply:
deepEquals compares object tags and prototypes. This is the cleanest fully spec compliant way I found to make an object with a null prototype for comparison. Putting __proto__ in the object literal apparently works, but only exists for legacy JS compat and is not truly standard. The other option of using object.create and specifiying all the members in there is real ugly as they all have to be property descriptors.